app/views/uploads/_data_criteria.html.erb

```diff
@@ -0,0 +1,126 @@
+  <% if rat_ref.is_a?(Hash) &&
+        (@specifics[@population_key] || @population_key == 'VAR')%>
+    <% rat_ref = rat_ref[:results].count > 0 %>
+    <% if @population_key != 'VAR' &&
+          should_switch_highlight?(reference, @population_key, @specifics, @rationale) %>
+      *<% rat_ref = !rat_ref %>
+    <% end %>
+  <% end %>
+  <span class="rationale eval-<%= rat_ref %>">
+<% end %>
+
+<% data_criteria = measure[:hqmf_document][:data_criteria][reference] %>
+<%# check attributes %>
+<% unless data_criteria %>
+  <% data_criteria = measure[:hqmf_document][:source_data_criteria][reference] %>
+<% end %>
+
+<% if data_criteria %>
+  <% if data_criteria[:subset_operators] && data_criteria[:subset_operators].length>0 %>
+    <% data_criteria[:subset_operators].each do |so| %>
+      <%= render partial: 'subset_operator', locals: {subset_operator:so, tag: "span"} %>
+    <% end %>
+  <% end %>
+
+
+  <% if data_criteria[:type] == 'derived' && (!data_criteria[:variable] || expand_variable) %>
+    <% if data_criteria[:definition] == 'satisfies_all' ||  data_criteria[:definition] == 'satisfies_any' %>
+      <%= render partial: 'satisfies', locals: {reference: reference, measure:measure, tag: "span"} %>
+    <% elsif data_criteria[:children_criteria] %>
+      <% if data_criteria[:derivation_operator] == 'INTERSECT' ||
+            data_criteria[:derivation_operator] == 'UNION' &&
+            !expand_variable %>
+        <ul><li><span class="set-op rationale-target rationale eval-<%= rat_ref %>">
+          <%= data_criteria[:derivation_operator] %> of:
+        </span></li>
+      <% end %>
+      <ul>
+        <% data_criteria[:children_criteria].each do |child| %>
+          <% if (data_criteria[:derivation_operator] == 'INTERSECT' ||data_criteria[:derivation_operator] == 'UNION') %>
+            <% if !expand_variable %>
+              <li class="set-op-item">
+            <% else %>
+              <li>
+            <% end %>
+          <% else %>
+            <li>
+              <span class="conjunction rationale-target"><%= (data_criteria[:derivation_operator] == 'XPRODUCT')? 'AND':'' %>: <span class="sr-only sr-highlight-status"></span></span>
+          <% end %>
+            <%= render partial: 'data_criteria', locals: {reference: child, expand_variable: false, measure:measure, tag: "span"} %>
+          </li>
+        <% end %>
+        <% if data_criteria[:temporal_references] %>
+          <% data_criteria[:temporal_references].each do |temp_ref| %>
+            <li>
+              <span class="rationale-target">
+                <%= render partial: 'temporal_reference', locals: {temporal_reference: temp_ref, measure:measure, tag: "span"} %>
+              </span>
+            </li>
+          <% end %>
+        <% end %>
+      </ul>
+      <% if data_criteria[:derivation_operator] == 'INTERSECT' ||
+            data_criteria[:derivation_operator] == 'UNION' &&
+            !expand_variable %>
```

Comment:
This 3-part conditional is in two places. Should just set to a variable so that it's only in one place.